index.js

```diff
@@ -19,7 +20,9 @@ console.log('hello world')
  * Example Output: n/a
  */
 
-// TODO: ADD CODE HERE
+var some_var = 3;
```

Comment:
It's good practice to use the newer `const` and `let` keywords instead of `var` because these statements have cleaner, safer scope and type checking.

Reply:
@elizabethompson, thank you so much for the quick response. I'm going to correct it now